### PR TITLE
(Update Interface) added additional compliments for /learn

### LINKED ISSUE #53365: Easy first pull request: add additional compliments for /learn UI
### Describe the Issue

We occasionally update our user interface to include new compliments to learners, which they see when they complete a coding challenge.

Here are some new pop-culture (video games / anime) reference compliments that we'd like to add to our existing roster of 100 or so messages.

Your mission: to find the right file to add these, and to add them to the right place without breaking the JSON formatting:

```
    "King of the Pirates!",
    "Thunder Breathing First Form!",
    "It's time to duel!",
    "You better believe it!",
    "Do a barrel roll!",
    "You can do this all day!",
    "It's super effective!",
```

I'll give you a hint for how you can find this file: one of the compliments is: "Everything's coming up Milhouse!"

Be sure to only modify the English JSON file. Open a pull request, we'll review it, and you can get your first accepted pull request.

### Affected Page

freecodecamp.org/learn

### Steps to Reproduce

Search the GitHub repo to find the correct file to modify. See description for a hint.

### Expected behavior

Compliments should be added to JSON file

### Screenshots

_No response_

### System

Any

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by SejuSpeaks**
Checklist:

- [ x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x ] My pull request targets the `main` branch of freeCodeCamp.
- [x ] I have tested these changes either locally on my machine, or GitPod.

Closes #53365

### CODE REVIEW

File: client/i18n/locales/swahili/motivation.json [hunk 1]
```diff
@@ -101,7 +101,14 @@
     "Cowabunga!",
     "Moon Prism Power!",
     "Plus Ultra!",
-    "Everything's coming up Milhouse!"
+    "Everything's coming up Milhouse!",
```
Comment (jdwilkin4) [inline client/i18n/locales/swahili/motivation.json:104]: It looks like you accidentally edited the wrong file.

Please revert the changes back to its original state here.

This is the correct english file that should be updated

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/i18n/locales/english/motivation.json

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes. 👍

**Comment by jdwilkin4** [comment]
Also, please make sure to update the PR template here to link the issue to this PR
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Screenshot 2024-01-25 at 6 15 37 PM

File: client/i18n/locales/swahili/motivation.json (hunk 1 above)
Comment (SejuSpeaks) [reply client/i18n/locales/swahili/motivation.json]: ok thank you for the kind help it should be alright now

**Review by jdwilkin4 — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
